testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetProxyIT.java

```diff
@@ -132,7 +132,8 @@ public void sortOrderIsRespected() throws Exception {
 
     List<String> receivedHashes = extractPackageHashes(entries);
 
-    assertThat(receivedHashes, is(sorted(receivedHashes)));
+    Collator dictionaryOrder = Collator.getInstance(Locale.ENGLISH);
+    assertThat(receivedHashes, is(Ordering.from(dictionaryOrder).sortedCopy(receivedHashes)));
   }
 
 
```

Comment:
sure, that's best addressed in a separate PR though given that `hashOrderedEntries` is not part of this PR and there are plenty of other tests that need cleaning up wrt. unused compiler warnings.